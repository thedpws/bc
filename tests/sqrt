sqrt(4)
sqrt(5)
