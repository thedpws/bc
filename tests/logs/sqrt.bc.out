2.00000000000000000000 2.23606797749978969640
